Sources/ProjectDescription/SourceFilesList.swift

```diff
@@ -15,13 +15,19 @@ public struct SourceFileGlob: ExpressibleByStringLiteral, Codable, Equatable {
     ///
     /// - Parameters:
     ///   - glob: Relative glob pattern.
+    ///   - excluding: Relative glob patterns for excluded files.
     ///   - compilerFlags: Compiler flags.
-    public init(_ glob: Path, excluding: Path? = nil, compilerFlags: String? = nil) {
+    public init(_ glob: Path, excluding: [Path]? = nil, compilerFlags: String? = nil) {
```

Comment:
wondering if default empty array would be better as it defaults to empty array in other's init

Reply:
Yep, lack of optionality in the case of the Array seems better